com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -562,6 +562,21 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
             if (rendererFeatures.Count != 0 && !isPreviewCamera)
                 ConfigureCameraColorTarget(m_ColorBufferSystem.PeekBackBuffer());
 
+            //setup RT descriptor for preview camera
+            if(isPreviewCamera)
```

Comment:
Not knowing a lot about the system, it seems flakey to me that we have to call m_ColorBufferSystem.SetCameraSettings specifically for a type of camera. Shouldn't we make sure this is properly configure for all types of cameras.

Looking at the code, it seems we call it here while creating the camera color render target: https://sourcegraph.com/github.com/Unity-Technologies/Graphics/-/blob/com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs?L1104

This implies to me that there could be more camera of paths that don't call that function and still need this setup. It seems this other PR is fixing a similar issue by forcing the preview camera to call create color render target, which also seems flakey to me. https://github.com/Unity-Technologies/Graphics/pull/6723/

Can the SetCameraSettings be something that is called while initializing each camera?

Reply:
I agree, I'll move it outside of the if( camera.type ...) statement so it's always set. The BufferSystem should only create targets when they are actually needed but there is no harm in setting the descriptor even if it might never be needed.